scripts/add_box.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+# Copyright 2011-2014, Michael Ferguson
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above
+#    copyright notice, this list of conditions and the following
+#    disclaimer in the documentation and/or other materials provided
+#    with the distribution.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+# "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+# LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+# FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+# COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+# INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+# BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+# LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+# LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+# ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+
+import argparse
+import rospy
+from moveit_python import PlanningSceneInterface
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        description="Remove objects from the MoveIt planning scene.")
+    parser.add_argument("--name",
+                        nargs="?",
+                        help="Name of object to remove")
+    args = parser.parse_args()
+
+    if args.name:
+        rospy.init_node("add_boxes")
+        scene = PlanningSceneInterface("/base_link")
+        print("Adding Box with name: %s" % args.name)
+        scene.addBox(args.name, 0.1,0.1,0.1, 3,3,3, True)
```

Comment:
It would be handy if the user could optionally specify the coordinates on the command line.
Obviously not required because this is mostly a test-script, but it would make a nice command line tool.

Reply:
I concur -- also made them executable - https://github.com/mikeferguson/moveit_python/commit/8717b0515757c071da49321d0d3e4fbedad107d7